src/goto-analyzer/goto_analyzer_parse_options.cpp

```diff
@@ -295,6 +296,34 @@ void goto_analyzer_parse_optionst::get_command_line_options(optionst &options)
         "call-stack-recursion-limit", cmdline.get_value("call-stack"));
       options.set_option("history set", true);
     }
+    else if(cmdline.isset("loop-unwind"))
+    {
+      options.set_option("local-control-flow-history", true);
+      options.set_option("local-control-flow-history-forward", false);
+      options.set_option("local-control-flow-history-backward", true);
+      options.set_option(
+        "local-control-flow-history-limit", cmdline.get_value("loop-unwind"));
+      options.set_option("history set", true);
+    }
+    else if(cmdline.isset("branching"))
```

Comment:
That is correct and in part why the combined option is known as `--loop-unwind-and-branching`.  I am reluctant to address this issue in this PR because it affects almost all options to goto-analyze`, all of the major options have an implicit ordering and check for absence but not duplication.